tests/queries/0_stateless/02935_date_trunc_case_unsensitiveness.sql

```diff
@@ -0,0 +1,9 @@
+SELECT dateTrunc('DAY', toDateTime('2022-03-01 12:55:55'));
```

Comment:
@yariks5s Please update the [docs](https://clickhouse.com/docs/en/sql-reference/functions/date-time-functions#date_trunc) of `date_trunc`